### DISCUSSION TITLE
E2 Eject with Position, Angle

### DISCUSSION THREAD

**Discussion by Denneisk** [wiremod/wire#2711] [Suggestions]
For functional parity with the Wire Vehicle Exit Point, have an overload for `entity:ejectPod()` that has position and angle arguments (`entity:ejectPod(vector pos, angle ang)`). Position argument sets the ejecting player's position to the vector, angle argument sets the ejecting player's angle.

**Comment by Hazeofdream** [comment]
me omw to ejecting people to the void

**Comment by vurvdev** [comment]
> me omw to ejecting people to the void

Exit point is clamped to 1000 units away, E2 should be clamped in the same way.

**Reply by Denneisk** [reply]
From the vehicle or the exit point itself?

**Reply by vurvdev** [reply]
Vehicle, otherwise you could just use the exit point as a tp itself
